core/session.ts

```diff
@@ -377,6 +302,15 @@ export class Session {
     return newTest;
   }
 
+  public notebook(target: dataform.ITarget): Notebook {
```

Comment:
should this not have the same/similar parameters as the other functions? (`declaration` is a bit special, i would consider it separately)